src/ConnectedMode.UnitTests/Install/ImportsBeforeFileGeneratorTests.cs

```diff
@@ -0,0 +1,163 @@
+﻿/*
```

Comment:
I don't know why GitHub is showing this a new set of tests; I just moved it and changed the namespace, exactly the same as the other test file i.e. no code changes.